### DISCUSSION TITLE
Integration tests on Windows

### DISCUSSION THREAD

**Discussion by Bananeweizen** [eclipse-tycho/tycho#432] [Q&A]
I'm still working on making my changes testable locally. Integration tests fail on my Windows machine. If I get it right, all the failing tests are related to building or including OS specific artifacts that are only available on other platforms. Failing tests are:

```
[ERROR] Errors:
[ERROR]   Tycho109ProductExportTest.exportPluginRcpApplication:91 » Verification Exit co...
[ERROR]   IncludeLaunchersTest.includeLaunchers:27 » Verification Exit code was non-zero...
[ERROR]   IncludeLaunchersTest.noIncludeLaunchers:40 » Verification Exit code was non-ze...
[ERROR]   JdtBreeJava8Test.testCompilerJdtBreeJava8:25 » Verification Exit code was non-...
[ERROR]   CustomSourceEncodingTest.test:27 » Verification Exit code was non-zero: 1; com...
[ERROR]   BaselineValidateAndReplaceTest.testBaselineFailCommon_newAttachedArtifact:186 » Verification
[ERROR]   MetaRequirementsTest.testProductInstallationWithCustomTouchpoint:32 » Verification
```
Should these tests work fine on Windows? If not, I would add something like `assumeTrue("Test uses OS specific launcher artifacts", System.getProperty("os.name").contains("linux"))` to them. However, I'm not really sure of that solution. Since MetaRequirementsTest depends on some MacOS specific stuff I would expect it to fail with the github action during normal builds. Since those builds pass, I must be missing something else, right?

**Comment by laeubi** [comment]
Could you enable full stacktraces and post details about the failure? The OS specific artifacts should build under any os, but there might be some issues regarding the interpretation of path names in the test assertions itself.
